### ISSUE TITLE
Styles dropdown doesn't have a tooltip

### ISSUE THREAD

**Issue by charlttsie** [ckeditor/ckeditor5#16129]
## 📝 Provide detailed reproduction steps (if any)

1. Open e.g. https://ckeditor.com/docs/ckeditor5/latest/examples/builds-custom/full-featured-editor.html
2. Hover over the Styles dropdown

### ✔️ Expected result

A tooltip should display

### ❌ Actual result

There's no tooltip:

https://github.com/ckeditor/ckeditor5/assets/132071257/1d96d26e-6612-416a-b7e1-4edbddfefdc3

## 📃 Other details

* Browser: Any
* Not a recent regression - the tooltip is also missing in older docs versions

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by narengit** [comment]
After expanding style dropdown, tooltip is not appearing for all the Style options.
In Safari browser Tooltip is coming and this behavior is specific to the Safari. tooltip is not coming in Edge and chrome browser.

**Comment by thomasmccarten** [comment]
> noTooltip.mov 
> After expanding style dropdown, tooltip is not appearing for all the Style options. In Safari browser Tooltip is coming and this behavior is specific to the Safari. tooltip is not coming in Edge and chrome browser.

Yeah, this is annoying.
